lib/tictactoe.rb

```diff
@@ -1,19 +1,37 @@
 require_relative "./display"
 require_relative "./presenter"
+require_relative "./board"
+require_relative "./player"
 
 class TicTacToe
-  def initialize(presenter, display)
+  def initialize(presenter, display, board, player)
     @presenter = presenter
     @display = display
+    @board = board
+    @player = player
+  end
+
+  def show_board
+    @display.output(@presenter.display_board(@board))
+  end
+
+  def play_turn
+    selection = @player.move
+    position = selection - 1
```

Comment:
Why do we need to decrement the selection? Should `TicTacToe` need to know this detail?

Reply:
We decrement to place the token in the correct array cell.  I thought, TicTacToe, as executioner, could handle this responsibility.  Maybe it's the player's move method?

Reply:
Yeah, it seems like the reason we need to decrement because the `Board` expects an index of the underlying array instead of a position on the board. Maybe this is an indication that we can push this responsibility into the board?

Reply:
okay